source/features/enable-file-links-in-compare-view.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import GitBranchIcon from 'octicon/git-branch.svg';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import GitHubURL from '../github-helpers/github-url';
+import {getCurrentBranch} from '../github-helpers';
+
```

Comment:
```suggestion

/** Rebuilds the "View file" link because it points to the base repo and to the commit, instead of the head repo and its branch */
```